OktaIdx.podspec

```diff
@@ -24,5 +24,5 @@ Integrate your native app with Okta using the Okta Identity Engine library.
   spec.source_files = 'Sources/OktaIdx/**/*.swift'
   spec.swift_version = "5.5"
 
-  spec.dependency "OktaAuthFoundation", "1.1.4"
+  spec.dependency "OktaAuthFoundation", "~> 1.1"
```

Comment:
I'm assuming this means the highest bugfix version of 1.1?

Reply:
CocoaPods is kinda weird. Essentially this says "Up to the next major version".  For more information: https://guides.cocoapods.org/using/the-podfile.html#specifying-pod-versions